docs/brand-guide.rst

```diff
@@ -0,0 +1,84 @@
+####################
+Solidity Brand Guide
+####################
+
+This brand guide features information on Solidity's brand policy and
+logo usage guidelines.
+
+The Solidity Brand
+==================
+
+The Solidity programming language is an open-source, community project
+governed by a core team. The core team is sponsored by the `Ethereum
+Foundation <https://ethereum.foundation/>`_.
+
+This document aims to provide information about how to best use the
+Solidity brand name and logo.
+
+We encourage you to read this document carefully before using the
+brand name or the logo. Your cooperation is highly appreciated!
+
+Solidity Brand Name
+===================
+
+"Solidity" should be used to refer to the Solidity programming language
+solely.
+
+Please do not use "Solidity":
+
+- To refer to any other programming language.
+
+- In a way that is misleading or may imply association of unrelated
+modules, tools, documentation, or other resources with the Solidity
+programming language.
```

Comment:
@franzihei for some reason the docs compiler doesn't like when a single item in a bullet list has a line break. So this should be all a single line.

Reply:
I think all it wants is not having the empty lines between the points.

Reply:
It also seems to be happy if you indent the new lines with 2 spaces.

Reply:
Ah ok, makes sense